doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,317 @@
+    "cassandraCompression":{
+        "org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraCompression":"NONE"
+    },
+    "cassandraExecuteRequestType":{
+        "org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraExecuteRequestType":"SYNC"
+    }
+}
+```
+
+## Administration
+
+The following Admin REST API call example illustrates how to create an instance of the Cassandra log appender.
+
+```bash
+curl -v -S -u devuser:devuser123 -X POST -H 'Content-Type: application/json' -d '{"pluginClassName":"org.kaaproject.kaa.server.appenders.cassandra.appender.CassandraLogAppender", "applicationId":"113", "applicationToken":"08690892651366457342", "jsonConfiguration":"{\"cassandraServers\" : [ { \"host\" : \"127.0.0.1\", \"port\" : 9042   } ],   \"cassandraCredential\" : { \"org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraCredential\" : {   \"user\" : \"user\",   \"password\" : \"password\" }   },   \"keySpace\" : \"kaa_logs\",   \"tableNamePattern\" : \"logs_$app_token_$config_hash\",   \"columnMapping\" : [ { \"type\" : \"CLIENT_FIELD\", \"value\" : {   \"string\" : \"applicationToken\" }, \"columnName\" : \"application_token\", \"columnType\" : \"BOOLEAN\", \"partitionKey\" : false, \"clusteringKey\" : false   }, { \"type\" : \"SERVER_FIELD\", \"value\" : {   \"string\" : \"applicationToken\" }, \"columnName\" : \"application_token\", \"columnType\" : \"BOOLEAN\", \"partitionKey\" : false, \"clusteringKey\" : false   } ],   \"clusteringMapping\" : [ { \"columnName\" : \"application_token\", \"order\" : \"DESC\"   } ],   \"cassandraBatchType\" : { \"org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraBatchType\" : \"UNLOGGED\"   },   \"cassandraSocketOption\" : { \"org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraSocketOption\" : {   \"sendBufferSize\" : { \"int\" : 8192   },   \"receiveBufferSize\" : { \"int\" : 8192   },   \"soLinger\" : { \"int\" : 0   },   \"connectionTimeout\" : { \"int\" : 5000   },   \"readTimeout\" : { \"int\" : 12000   },   \"reuseAddress\" : { \"boolean\" : false   },   \"keepAlive\" : { \"boolean\" : false   },   \"tcpNoDelay\" : { \"boolean\" : false   } }   },   \"executorThreadPoolSize\" : 1,   \"callbackThreadPoolSize\" : 2,   \"dataTTL\" : 0,   \"cassandraWriteConsistencyLevel\" : { \"org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraWriteConsistencyLevel\" : \"ONE\"   },   \"cassandraCompression\" : { \"org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraCompression\" : \"NONE\"   },   \"cassandraExecuteRequestType\" : { \"org.kaaproject.kaa.server.appenders.cassandra.config.gen.CassandraExecuteRequestType\" : \"SYNC\"   } }", "description":"New sample Cassandra log appender", "headerStructure":["KEYHASH", "TIMESTAMP"], "name":"New Cassandra DB appender", "maxLogSchemaVersion":2147483647, "minLogSchemaVersion":1, "tenantId":"10"}' "http://localhost:8080/kaaAdmin/rest/api/logAppender" | python -mjson.tool
```

Comment:
This is ugly. Please use -f option and specify file name. We can provide a content of the file  next to the api call